packit/constants.py

```diff
@@ -91,3 +91,47 @@
     "This repository is maintained by packit.\nhttps://packit.dev/\n"
     "The file was generated using packit {packit_version}.\n"
 )
+
+# flake8: noqa
+# ignoring whole file with flake b/c some of the lines in the constant below are too long
+# since that code is lua+bash it's impossible to wrap those lines
+
+# shell code wrapped in lua as a python constant, what could go wrong?
+# for real now: this horror is being used when creating source-git repos from upstream
+# packit creates a file ~/.rpmmacros and adds this as a content to be able to run
+# `rpmbuild -bp` and get a git repo out of %prep
+# once the command is finished, the file is cleaned up
+RPM_MACROS_FOR_PREP = r"""
```

Comment:
What about storing it as a raw (lua) file?

(The checking of the syntax and doing updates might be easier.)

Reply:
Yes, we could do that, though - where would we store it? How would the file get installed from a python package?

This is easier to do in terms of delivery though harder for doing any changes to that file.

Reply:
Via `data_files`/`package_data`? But I agree that it takes some time to do that right.

Reply:
I'll add this to the tomorrow arch session